File: src/main/java/me/ling/kipfin/vkbot/utils/BroadcastMessage.java
package me.ling.kipfin.vkbot.utils;

import org.jetbrains.annotations.Nullable;

import java.util.List;

public class BroadcastMessage {

    private final String message;
    private final List<Integer> users;
    @Nullable
    private final String image;

    public BroadcastMessage(String message, List<Integer> users, @Nullable String image) {
        this.message = message;
        this.users = users;
        this.image = image;
    }

    public String getMessage() {
        return message;
    }

    public List<Integer> getUsers() {
        return users;
    }

    @Nullable
    public String getImage() {
        return image;
    }
}

File: src/main/java/me/ling/kipfin/vkbot/entities/message/BroadcastMessage.kt
package me.ling.kipfin.vkbot.entities.message

import com.vk.api.sdk.objects.messages.Keyboard
import java.io.File

class BroadcastMessage(text: String, keyboard: Keyboard? = null, file: File? = null, val users: List<Int>) : ImageMessage(text, keyboard, file)